gallery/visualtests/nxMultiFileUpload.js

```diff
@@ -0,0 +1,545 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+const fs = require('fs');
+const path = require('path');
+const tmp = require('tmp-promise');
+
+const { setupBrowser } = require('./testUtils');
+
+async function fillFile(path, numBytes) {
```

Comment:
This is duplicated from the other file.  Move it to `testUtils` for a shared implementation.

Reply:
> This is duplicated from the other file. Move it to `testUtils` for a shared implementation.

Moved into `testUtils` here: [9af2ab1](https://github.com/sonatype/sonatype-react-shared-components/pull/850/commits/9af2ab1f211abf5599fffcf9e1624c07875d7e75)